client/scripts/controllers/server/threads.ts

```diff
@@ -36,8 +37,13 @@ export const modelFromServer = (thread) => {
 
 class ThreadsController {
   private _store = new ProposalStore<OffchainThread>();
+  private _listingStore = new ProposalStore<OffchainThread>();
```

Comment:
Can we add a comment/comments here to explain what each store does? It doesn't seem obvious from their names